static/js/request_button.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Displays the response from the request below the button.
+ * @param {object} response - The response from the endpoint call. 
+ * @param {HTMLElement} button - The request button for the current example.
+ */
+function displayResult(response, button) {
+  let responseDiv = document.createElement('div');
+  responseDiv.dataset.id = button.id;
+
+  let responseHeader = document.createElement('h4');
+  responseHeader.innerText = 'Live Response';
+
+  let statusPre = document.createElement('pre');
+  statusPre.className = 'headers';
+
+  let statusCode = document.createElement('code');
+  statusCode.innerText = 'Status: 200 OK';
+  statusPre.appendChild(statusCode);
+
+  let responsePre = document.createElement('pre');
+  responsePre.className = 'body-response';
+
+  let responseCode = document.createElement('code');
+  responseCode.className = 'language-javascript';
+  responseCode.innerText = JSON.stringify(response, null, 2);
+  responsePre.appendChild(responseCode);
+
+  responseDiv.appendChild(responseHeader);
+  responseDiv.appendChild(statusPre);
+  responseDiv.appendChild(responsePre);
+
+  button.parentNode.parentNode.insertBefore(responseDiv, button.parentNode.nextSibling.nextSibling);
```

Comment:
I think we need to change this. This relies, imo, too heavily on the organization of the HTML elements, and if we were to change that I think we'll encounter so issues. Would it be possible to use some class/id instead?

Reply:
This function was not longer necessary and has been removed